src/MonologExtraBundle.php

```diff
@@ -4,6 +4,6 @@
 
 use Symfony\Component\HttpKernel\Bundle\Bundle;
 
-class TheRedDotMonologExtraBundle extends Bundle
+class MonologExtraBundle extends Bundle
```

Comment:
This change will introduce a BC break.

Maybe this should be extracted and reserved for 5.x.

Reply:
strange I do not remember making this change